### PR TITLE
Font styles

### PR DESCRIPTION
**Pull request by Rankz**

### CODE REVIEW

File: frontend/Screens/UserProfile.tsx [hunk 1]
```diff
@@ -43,17 +47,35 @@ const styles = StyleSheet.create({
...
   const numberOfMunroesClimbed = mountainList
     .filter((mountain) => mountain.Statuses[0]?.climbed).length;
 
+  // eslint-disable-next-line prefer-const
+  let munroeClimbedPercentage = (numberOfMunroesClimbed / totalMunroes) * 100;
+  // eslint-disable-next-line prefer-const
+  let progressBarPercentageNum = Math.round(munroeClimbedPercentage);
+  // eslint-disable-next-line prefer-const
+  let progressBarPercentage = `${progressBarPercentageNum}%`;
+
```
Comment (jonbrook) [inline frontend/Screens/UserProfile.tsx:78]: why are these let? they are not re-assigned so should be const's.

Comment (Rankz) [reply frontend/Screens/UserProfile.tsx:78]: These are 'let' becuase number of munroes climbed can change at anytime, which will change the all the other numbers. Have I got the right idea?

Comment (jonbrook) [reply frontend/Screens/UserProfile.tsx:78]: Nope, for example on row 73, the assignment is to the calculation of (a / b) * 100. The resulting value of that can change but we are never reassigning the variable to anything else, it always remains the (a / b) * 100.

Comment (Rankz) [reply frontend/Screens/UserProfile.tsx:78]: Ahhhhh, I'm with you

**Review by jonbrook — APPROVED** [review]
LGTM :tada:

**Pull request merged**